rake/spec/default/60-st2_all-services-ok_spec.rb

```diff
@@ -9,17 +9,29 @@
     subject { host(spec[:rabbitmqhost]) }
     it { is_expected.to be_reachable }
   end
+  describe 'rabbitmq' do
+    subject { host(spec[:rabbitmqhost]) }
+    it { should be_reachable.with( :port => 5672, :timeout => 1 ) }
```

Comment:
examples can have multiple tests, so above should be merge together/

either 
```ruby
   describe 'mongodb' do
      subject { host(spec[:mongodbhost]) }
      it { is_expected.to be_reachable }
      it { is_expected.to be_reachable.with :port => 27017, :timeout => 1 }
    end
```

or 
```ruby
   describe 'mongodb' do
      subject { host(spec[:mongodbhost]) }
      it { is_expected.to be_reachable.with :port => 27017, :timeout => 1 }
    end
```

PS. use defualt syntax (not should syntax)